### DISCUSSION TITLE
What does it cost to store files (audio)

### DISCUSSION THREAD

**Discussion by redimongo** [ipfs/ipfs#498] [Poll]
So I am looking into IPFS as a storage tool for an audio / video project we are launching in 2023. We are very happy to host the node and provide all the storage requirements for our members to use. But if we are the Node that users are uploading to. Then there should not be any cost to us? As we own the node that is running - Am I understanding this correctly?

We see that IPFS could assist us in allowing us to store copies of the modified file once we insert an ad into the file. This would allow us to prove to an advertiser that their ad was embedded into the audio on delivery of the file.

If anyone knows the answer I look forward to hearing your opinion and views.

**Comment by 2color** [comment]
The [IPFS Forum](https://discuss.ipfs.tech) is a better place to have these discussions
